pom.xml

```diff
@@ -3,7 +3,7 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.benetech</groupId>
+	<groupId>org.benetech</groupId>
```

Comment:
you should change the java package used across the application to match this

Reply:
Right